compiler/crates/relay-compiler/src/build_project/mod.rs

```diff
@@ -34,7 +34,7 @@ pub use generate_artifacts::{
 };
 use generate_extra_artifacts::generate_extra_artifacts;
 use graphql_ir::Program;
-use graphql_transforms::FB_CONNECTION_INTERFACE;
+use graphql_transforms::CONNECTION_INTERFACE;
```

Comment:
Figured it wasn't really a callsite's concern which connection it should pull in.